index.html

```diff
@@ -56,6 +56,210 @@ <h1 class="hero_title">I'm Reza.<br>Glad to see you!</h1>
                 
             </div>
         </section>
+        <section class="project-grid">
+            <div class="project-card">
+                <a href="" class="project-img"><img src="./works/project01.png" /></a>
+                <h2>Tonic</h2>
+                <div class="project-overhead">
+                    <ul class="overhead">
+                        <li class="overhead-1">Canopy</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="project-detail">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="built-with">
+                    <li>
+                        <button class="build">HTML</button>
+                    </li>
+                    <li>
+                        <button class="build">CSS</button>
+                    </li>
+                    <li>
+                        <button class="build">Javascript</button>
+                    </li>
+                </ul>
+                <button class="see">See Project</button>
+            </div>
```

Comment:
- [ ] Let's make this match the Figma design in order to show your skills with basic HTML and custom CSS :smile: :muscle: 
This is what we have when compared with the Figma's version:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/243119907-9070385c-de99-44b6-ad9c-bbaf8c3e7fef.png)|![image](https://user-images.githubusercontent.com/99715640/243120013-bc9e7a5d-5f01-4384-bcea-c057177f3e60.png)|

### To Improve:
- [ ] The seperators between your `Canopy`, `Back end, and `date` seem to need adjustment so that it doesn't collapse into other parts of the page, making it difficult to read them.
- [ ] The list of technologies also appears to need some uniform spacing to distinguish one from the other.
- [ ] Like I mentioned in the `About me`, your buttons need to show different styles at different states, and getting the text centered will improve your design a lot.